docs/source/cli/graph-construction/distributed/gspartition/ec2-clusters.rst

```diff
@@ -64,7 +64,7 @@ Collect the IP address list
 ...........................
 The GraphStorm Docker containers use SSH on port ``2222`` to communicate with each other. Users need to collect all IP addresses of all the instances and put them into a text file, e.g., ``/data/ip_list.txt``, which is like:
 
-.. figure:: ../../../../../tutorial/distributed_ips.png
+.. figure:: ../../../../../../tutorial/distributed_ips.png
```

Comment:
Do we have a preview page for this ip page?

Reply:
Yes. This PR's preview of this page: https://james4graphstorm.readthedocs.io/en/james_reorg_doc/cli/graph-construction/distributed/gspartition/ec2-clusters.html#setup-the-ip-address-file-and-check-port-status